requirements/common.txt

```diff
@@ -109,3 +109,6 @@ httplib2==0.9.1
 # Required by django.contrib.migrations
 # sha256: Z4xsNspLAUBRd9qLhO7PkuySyfbHYjlsllu10wXyD4E
 sqlparse==0.1.16
+
+# sha256: hOvuRduDR6irx3nZHsBFro5mNLS4uCxvu9yKHxDv2Vw
+memoize==1.0.0
```

Comment:
I'm not sure it's worth to add a dependency for 5 lines of code... Can we embed those 5 lines somewhere in treeherder?